src/mlpack/methods/ann/layer/batch_norm.hpp

```diff
@@ -138,6 +138,12 @@ class BatchNorm
   //! Get the variance over the training data.
   OutputDataType TrainingVariance() { return runningVariance / count; }
 
+  //! Get the number of input units.
+  size_t const& InSize() const { return size; }
```

Comment:
See comment above, I think we should use the same style across the codebase.

Reply:
Okay! Will update these changes in a commit probably a few hours from now.